compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -1226,7 +1226,13 @@ LogicalResult initGPULaunchConfig(ModuleOp moduleOp) {
     }
 
     if (!rootOperation) {
-      // No root operation found. Allow it to pass through without a config.
+      // No root operation found, set it to default.
+      auto translationInfo = IREE::Codegen::TranslationInfoAttr::get(
+          funcOp.getContext(),
+          IREE::Codegen::DispatchLoweringPassPipeline::LLVMGPUDefault);
+      if (failed(setTranslationInfo(funcOp, translationInfo))) {
+        return failure();
+      }
```

Comment:
What would a dispatch without a root operation be? I thought copy only dispatches should be bufferized to materialize a copy; would this just be cases that we don't know how to handle?

Reply:
Actually, is this trying to solve a specific case? Is there a missing test here?

Reply:
It is not tested as a pipeline test, but if there are no root ops, it just does a pass through. Copy only dispatches get intercepted ahead of time and get bufferized before we get here.